decidim-system/app/views/decidim/system/admins/edit.html.erb

```diff
@@ -1,11 +1,11 @@
 <% provide :title do %>
-  <h2><%= t ".title" %></h2>
+  <h1 class="h1"><%= t ".title" %></h1>
```

Comment:
just a curiosity ... why do we have a css class called H1, that we assign on H1 tag? is not much simpler to just define the rules on the tag?

Reply:
Initially the idea was provide a different style for each of the headings the design had defined... but that weren't a11y-friendly, as not all sites have the same heading order.

For that same reason, is not a good idea to set defaults for the plain tags, instead of using CSS classes, as long as your site have these peculiarities.

We're aware of that curiosity / missunderstanding, and we've planned to rename such classes to headingXL, headingLG, or similar... in future issues

Reply:
Great, Thanks!